### ISSUE TITLE
I created a CLEANED dataset that combines cases, recoveries, and deaths to 1 CSV file

### ISSUE THREAD

**Issue by jbarton311** [CSSEGISandData/COVID-19#1241]
This is not an issue perse, but I wanted to share this info with the community.

### I've created a SINGLE, cleaned dataset that includes ALL cases, recoveries, and deaths into a single CSV file. It also:
- Contains data in a more "stacked" format and is rolled up by `state`, `country`, and `date`.
- Cleans all historical US data to roll up at the state level.
- Adds several additional columns for easier analysis.

I believe it is in an easier to use format than the CSVs posted to this repo (especially when performing analysis with BI tools).

The dataset along with more info (and the code used to create it) can be found [here](https://github.com/jbarton311/coronavirus-analysis/tree/master/coronavirus/output_data)

Thank you JHU for all of the hard work here!

**Comment by valeriupredoi** [comment]
very cool! I am already using JHU data for linear analyses [here](https://github.com/valeriupredoi/COVID-19_LINEAR) - might switch to yours soon if that's okay with you :beer:

**Comment by rufuspollock** [comment]
Cool. We did this a couple of weeks ago for Open Data Day and repo is here https://github.com/datasets/covid-19 - maybe we can join forces?

**Comment by jbarton311** [comment]
I should have also mentioned - I have leveraged this dataset to build a [simple dashboard](https://datastudio.google.com/reporting/62126a88-78d5-48d0-8fa6-6d2c188803e6) using Google Data Studio.

**Comment by paolinic03** [comment]
Hey @jbarton311 so when you say the dataset contains all recoveries, why does your simple dashboard show no recoveries? Not seeing the difference. Thanks.

**Comment by jbarton311** [comment]
Hi @paolinic03 ,

There are currently issues with the JHU data for US recoveries. Once those are fixed, the dashboard will display proper US recoveries data. [Here](https://github.com/jbarton311/coronavirus-analysis/tree/master/coronavirus/output_data#california-sample-data) is a sample of what data is contained in my dataset. You'll see several `recoveries` columns.

**Comment by paolinic03** [comment]
@jbarton311 ok makes sense. Appreciate it. Good job with your dashboard. Like the look.

**Comment by jbarton311** [comment]
FYI - based on the [recent announcement from JHU,](https://github.com/CSSEGISandData/COVID-19/issues/1250) I will have to modify what I've done to match their new data moving forward. Output data format will have to change slightly.

**Comment by joshp112358** [comment]
I made this based on the dataset. It does some basic times series visualisations.

https://joshyp.shinyapps.io/COVID_VIZ/

**Comment by verayanakieva** [comment]
Hi,

Do you plan to continue updating the data (combined incl. recovered)?

https://datahub.io/core/covid-19#data

It'd be great if that's the case.

**Comment by jbarton311** [comment]
@verayanakieva 

I plan to continue to update the data but am waiting for JHU to come out with the US specific datasets that they mentioned they'd be releasing. I will not being including recovered as they are no longer tracking that metric.

**Comment by valeriupredoi** [comment]
> @verayanakieva
> 
> I plan to continue to update the data but am waiting for JHU to come out with the US specific datasets that they mentioned they'd be releasing. I will not being including recovered as they are no longer tracking that metric.

I think the changes are out since yesterday's daily dataset, mate - I had to change my data module in my code since since yesterday there's heaps more on individual cities in the US and also a total restructuring of the order in which members are in the table :beer:

**Comment by rufuspollock** [comment]
> Do you plan to continue updating the data (combined incl. recovered)?
> 
> https://datahub.io/core/covid-19#data

Yes, we do. The github source is here https://github.com/datasets/covid-19

**Comment by jbarton311** [comment]
@valeriupredoi - I am planning on waiting for them to release the time series CSVs for US which hopefully will be very soon. From what I hear, the county data in the daily CSVs only goes back so far.

@rufuspollock - where are you sourcing recoveries data from? Is it a reliable source of data? As I understood, this metric was not reliably tracked.

**Comment by josebsalazar** [comment]
> Cool. We did this a couple of weeks ago for Open Data Day and repo is here (details in https://www.datopian.com/blog/2020/03/17/odd-covid-19/):
> 
> https://github.com/datasets/covid-19
> 
> Maybe we can join forces?
> 
> There's also json data from DataHub dataset we are publishing here https://datahub.io/core/covid-19#data



> > Do you plan to continue updating the data (combined incl. recovered)?
> > https://datahub.io/core/covid-19#data
> 
> Yes, we (@datasets / @datopian) plan to keep that updated.
> 
> BTW the github source is here https://github.com/datasets/covid-19

Thanks, i was going to write my own python script, but I will leverage your work.
